### DISCUSSION TITLE
Pass ripgrep arguments to rga-fzf

### DISCUSSION THREAD

**Discussion by deepdishpep** [phiresky/ripgrep-all#218] [Q&A]
When I use `rga` I can pass the argument `--no-ignore` which gets used by `ripgrep`. Is there any way to add this as an argument when using `rga-fzf`?

**Comment by Konfekt** [comment]
You can adapt `$FZF_DEFAULT_COMMAND`, for example

```sh
env FZF_DEFAULT_COMMAND="rga --no-ignore" rga-fzf
```

Have a look at my [command-line bindings](https://github.com/Konfekt/fzf-grep-all.sh) for inspiration
